cdap-api-common/src/main/java/co/cask/cdap/api/data/format/Formats.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2015-2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.data.format;
+
+/**
+ * Names of {@link RecordFormat RecordFormats} built into the platform.
+ */
+public final class Formats {
```

Comment:
can we keep track of this breaking change to update docs

Reply:
All these are part of incompatible change due to removal of Stream.